src/feedback/trulens/feedback/llm_provider.py

```diff
@@ -1314,14 +1313,43 @@ def stereotypes_with_cot_reasons(
 
         return self.generate_score_and_reasons(system_prompt, user_prompt)
 
+    def _remove_trivial_statements(self, statements: List[str]) -> List[str]:
```

Comment:
Would be good to add any benchmarking results post this change. @sfc-gh-dhuang pls help with that.

Reply:
Yes can do. Time to start a backlog of things that we wanna meta-evaluate later after the PrPr :D